README.md

```diff
@@ -1,101 +1,106 @@
+In the `src` folder of the starter kit, you'll find three JavaScript files:
 
-> **GUI**: Go to your AWS Lambda console, select the function associated your Lambda environment, and import the `dist.zip` file. After you upload it, there should now be a minified `index.js` file located inside of your Lambda function's "Code Source" section.
+- `src/viewer_request.js` contains sample code that fetches and caches the client configuration, initializes the Kameleoon SDK with this configuration, and modifies the outgoing request by adding the User ID cookie and feature flag value to the header.
+- `src/viewer_response.js` file contains sample code that fetches the User ID cookie from the incoming request and attaches the cookie to the outgoing response.
+- `src/helpers.js` contains some additional platform-specific code that demonstrates common features of the Kameleoon SDK.
 
-> **CLI**: You can use the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to update your AWS Lambda function programmatically. Example command: `aws lambda update-function-code --function-name my-aws-lambda-at-edge-function --zip-file fileb://dist.zip`.
+You'll use these files to initialize your environment:
 
-> **Lambda Layers**: If you need additional libraries, custom runtimes, or configuration files to use alongside your Lambda function, consider looking into utilizing [Lambda Layers](https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html).
+1. Navigate to `src/viewer_request.js` and update the `YOUR_SITE_CODE` and `YOUR_FEATURE_KEY` values with your [Kameleoon credentials](https://help.kameleoon.com/api-credentials).
+1. Review or adjust the feature flags or experiments you've configured with the Kameleoon NodeJS SDK, and hook into the lifecycle events by inserting your desired logic into the switch-case statements in `src/viewer_request.js` and `src/viewer_response.js`.  For example, you can change headers, cookies, and more.
+1. Run `npm run build:viewer_request` to use Rollup to bundle the source code of `src/viewer_request.js` into a `dist/dist.zip` file that you'll import into Lambda. The `dist.zip`` file should be roughly **~49kb** in size for `src/viewer_request.js` assuming you have not made any additional changes.
+1. Upload the request `dist/dist.zip` file into Lambda using one of these options:
+    -**GUI**: Go to your AWS Lambda console, select the function associated with your Lambda environment, and import the `dist.zip` file. After you upload it, there should now be a minified `index.js` file located inside of your Lambda function's **Code Source** section.
+    **CLI**: You can use the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to update your AWS Lambda function programmatically. Example command: `aws lambda update-function-code --function-name my-aws-lambda-at-edge-function --zip-file fileb://dist.zip`.
```

Comment:
these two GUI and CLI are not starting from new line in view mode

Reply:
should be fixed now.